routing/filter_creation_metrics.go

```diff
@@ -0,0 +1,103 @@
+package routing
+
+import (
+	"time"
+
+	"github.com/zalando/skipper/filters"
+)
+
+const (
+	maxAge              = 2
+	metricsPrefix       = "routeCreationTime."
+	defaultCreationTime = "default"
+)
+
+type FilterCreationMetrics struct {
+	metrics   filters.Metrics
+	configIds map[string]map[string]int
```

Comment:
what's the goal of this cache here? Can you give an explanation?

Reply:
I added the documentation to RouteCreationMetrics

// RouteCreationMetrics reports metrics about the time it took to create metrics.
// It looks for filters of type OriginMarker to determine when the source object (e.g. ingress) of the route
// was created.
// If an OriginMarker with the same type and id is seen again, the creation time is not reported again, because
// a route with the same configuration already existed before.